src/goose/toolkit/developer.py

```diff
@@ -67,7 +72,7 @@ def update_plan(self, tasks: List[dict]) -> List[dict]:
         table.add_column("Status", justify="left")
 
         # Mapping of statuses to emojis for better visual representation in the table.
-        emoji = {"planned": "⏳", "complete": "✅", "failed": "❌", "in-progress": "🕓"}
+        emoji = {"planned": "", "complete": "", "failed": "❌", "in-progress": ""}
```

Comment:
Accidental drop of these emoji?

Reply:
ah yes, goose really likes to mess those up...